src/parser/Class_Parser.ii

```diff
@@ -0,0 +1,658 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   Class_Parser.ii
```

Comment:
This is a non standard file extension (at least for Draco).  Why are you introducing it?  How is it different from existing file extensions?

Reply:
Are you no longer using .ii for implementation details of templates? Should I move the implementation into the .hh file?

Reply:
We use `.i.hh` for implementation details (to separate implementation from the design captured in the .hh` file.  Include the `.i.hh` form the bottom of the `.hh` file).  Use `.t.hh` for explicit template instantiation.  Ref: https://rtt.lanl.gov/redmine/projects/draco/wiki/Coding_Standards

Reply:
Ah. Will do.